macOS/DuckDuckGo/PrivacyDashboard/View/PrivacyDashboardViewController.swift

```diff
@@ -318,6 +318,16 @@ extension PrivacyDashboardViewController {
         return webVitalsResult
     }
 
+    private func isPirEnabledAndUserHasProfile() async -> Bool {
+        let isPIRFeatureEnabled = try? await Application.appDelegate.subscriptionAuthV1toV2Bridge.isEnabled(feature: .dataBrokerProtection)
+        guard let isPIRFeatureEnabled,
+              isPIRFeatureEnabled == true else {
+            return false
+        }
+        let profile = try? DataBrokerProtectionManager.shared.dataManager?.fetchProfile()
+        return profile != nil
+    }
+
```

Comment:
perhaps both checks are not needed?

Reply:
I think they might technically not be at the moment, but I wouldn't rule out us at some point deciding to keep the data in the DB even if DBP is disabled. So I think it's better with both checks